Node.js docs: `tool.execution_complete` is documented with a `toolName` field it does not have

### Summary

The Node.js extension-authoring docs state that the `tool.execution_complete` session event's `data` carries a `toolName` field. That event has no `toolName`, so a handler that reads `event.data.toolName` on completion gets `undefined`. `toolName` is emitted on `tool.execution_start`; a completion is correlated to its start by `toolCallId`.

Because the documented snippets are JavaScript, following the docs produces no error at all - the value is silently `undefined`.

### Where

Two event-field reference tables and three handler code-comments:

- `nodejs/docs/agent-author.md` - the "Key Event Types" table row for `tool.execution_complete`, and the comment in the `session.on("tool.execution_complete", ...)` example.
- `nodejs/docs/examples.md` - the "Top 10 Most Useful Event Types" table row for `tool.execution_complete`, and two completion-handler comments.

For example, in `nodejs/docs/agent-author.md`:

| Event | Key Data Fields |
| --- | --- |
| `tool.execution_complete` | `toolCallId`, `toolName`, `success`, `result`, `error` |

```js
session.on("tool.execution_complete", (event) => {
    // event.data.toolName, event.data.success, event.data.result
});
```

### Why it is wrong

The shipped types and the bundled event schema agree that the field does not exist on this event:

- `ToolExecutionCompleteData` has `toolCallId`, `success`, `result?`, `error?`, `model?`, `interactionId?`, `turnId?`, `toolTelemetry?`, ... but no `toolName`. `ToolExecutionStartData` has `toolName: string`.
- The bundled `session-events.schema.json` defines `ToolExecutionCompleteData` with `"required": ["toolCallId", "success"]` and `"additionalProperties": false`, and `toolName` is not among its properties - so it is not part of the event's contract. `ToolExecutionStartData` lists `toolName` in its `required` array.

This is also inconsistent within the repository: the shared event reference at `docs/features/streaming-events.md` already documents `tool.execution_complete` **without** `toolName`, and describes `toolCallId` as "Matches the corresponding `tool.execution_start`". The two Node.js pages are the only place that makes the claim - no other language binding's documentation does.

`nodejs/docs/examples.md` already shows the correct pattern: it reads `toolName` in `tool.execution_start` handlers, records the `toolCallId`, and matches the completion event by that id. It also states the rule directly: "Correlate `tool.execution_start` / `tool.execution_complete` events by `toolCallId`".

### Reproduction

```bash
npm install @github/copilot-sdk
```

```ts
import type {
    ToolExecutionCompleteData,
    ToolExecutionStartData,
} from "@github/copilot-sdk";

declare const complete: ToolExecutionCompleteData;
declare const start: ToolExecutionStartData;

start.toolName; // ok
complete.toolName;
//       ^^^^^^^^ error TS2339: Property 'toolName' does not exist on type 'ToolExecutionCompleteData'.
```

At runtime, inside a `tool.execution_complete` handler, `event.data` has own properties `toolCallId`, `success`, `result`, `model`, `interactionId`, `turnId` and `toolTelemetry`, and `Object.prototype.hasOwnProperty.call(event.data, "toolName")` is `false`, while `event.data.toolCallId` is set and matches the earlier `tool.execution_start`.

### Expected

The `tool.execution_complete` reference rows and completion-handler comments should not list `toolName`. To get the tool name in a completion handler, correlate by `toolCallId` with the earlier `tool.execution_start` event.

### Notes

The claim dates back to the commit that first added both pages (https://github.com/github/copilot-sdk/pull/737); the completion payload has not carried `toolName` in any released schema, including the one current at that time. It appears to have been copied from the adjacent `tool.execution_start` row.

It has gone unnoticed because nothing validates these snippets: `scripts/docs-validation` only scans the top-level `docs/` directory, so `nodejs/docs/**` is never extracted or compiled.

### Environment

- `@github/copilot-sdk` on `main`, with bundled `@github/copilot` 1.0.77
- Node.js 22, Linux x64

--- a/nodejs/docs/agent-author.md
+++ b/nodejs/docs/agent-author.md
@@ -258,7 +258,7 @@ Subscribe to session events. Returns an unsubscribe function.
 
 ```js
 const unsub = session.on("tool.execution_complete", (event) => {
-    // event.data.toolName, event.data.success, event.data.result
+    // event.data.success, event.data.result
 });
 ```
 
@@ -268,7 +268,7 @@ const unsub = session.on("tool.execution_complete", (event) => {
 | ------------------------- | ------------------------------------------------------ |
 | `assistant.message`       | `content`, `messageId`                                 |
 | `tool.execution_start`    | `toolCallId`, `toolName`, `arguments`                  |
-| `tool.execution_complete` | `toolCallId`, `toolName`, `success`, `result`, `error` |
+| `tool.execution_complete` | `toolCallId`, `success`, `result`, `error`             |
 | `user.message`            | `content`, `attachments`, `source`                     |
 | `session.idle`            | `backgroundTasks`                                      |
 | `session.error`           | `errorType`, `message`, `stack`                        |

--- a/nodejs/docs/examples.md
+++ b/nodejs/docs/examples.md
@@ -368,7 +368,7 @@ session.on((event) => {
 
 ```js
 const unsubscribe = session.on("tool.execution_complete", (event) => {
-    // event.data.toolName, event.data.success, event.data.result, event.data.error
+    // event.data.success, event.data.result, event.data.error
 });
 
 // Later, stop listening
@@ -417,7 +417,7 @@ session.on("assistant.message", (event) => {
 | `assistant.message`         | Agent's final response                           | `content`, `messageId`, `toolRequests`                 |
 | `assistant.message_delta`   | Message content chunks (ephemeral)               | `deltaContent`                                         |
 | `tool.execution_start`      | A tool is about to run                           | `toolCallId`, `toolName`, `arguments`                  |
-| `tool.execution_complete`   | A tool finished running                          | `toolCallId`, `toolName`, `success`, `result`, `error` |
+| `tool.execution_complete`   | A tool finished running                          | `toolCallId`, `success`, `result`, `error`             |
 | `user.message`              | User sent a message                              | `content`, `attachments`, `source`                     |
 | `session.idle`              | Session finished processing a turn               | `backgroundTasks`                                      |
 | `session.error`             | An error occurred                                | `errorType`, `message`, `stack`                        |
@@ -677,6 +677,6 @@ session.on("assistant.message", (event) => {
 });
 
 session.on("tool.execution_complete", (event) => {
-    // event.data.success, event.data.toolName, event.data.result
+    // event.data.success, event.data.result
 });
 ```

